src/app/beer_garden/garden.py

```diff
@@ -35,30 +35,13 @@ def get_gardens() -> List[Garden]:
     return db.query(Garden)
 
 
-def update_garden(garden_name: str, patch: PatchOperation) -> Garden:
-    """Applies updates to an Garden.
+def update_garden_config(garden: Garden):
+    db_garden = db.query_unique(Garden, id=garden.id)
+    db_garden.connection_params = garden.connection_params
+    db_garden.connection_type = garden.connection_type
 
-    Args:
-        garden_name: The Garden Name
-        patch: Patch definition to apply
-
-    Returns:
-        The updated Garden
-    """
-    garden = None
-
-    for op in patch:
-        operation = op.operation.lower()
-
-        if operation in ["initializing", "running", "stopped", "block"]:
-            garden = update_garden_status(garden_name, operation.upper())
-        elif operation == "heartbeat":
-            garden = update_garden_status(garden_name, "RUNNING")
-
-        else:
-            raise ModelValidationError(f"Unsupported operation '{op.operation}'")
-
-    return garden
+    update_garden(db_garden)
+    return db_garden
```

Comment:
Every function in this module should be returning the result of the update:

```python
return update_garden(db_garden)
```

Reply:
Fixed